src/components/pages/AddStaff/AddStaffForm.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import uuid from 'uuid';
+import {
+  Container,
+  Button,
+  Grid,
+  Header,
+  Segment,
+  Form,
+  Message,
+  Checkbox
+} from 'semantic-ui-react';
+
+const AddStaff = props => {
+  const { state, handleSubmit, handleChange, checkboxChange } = props;
+  setTimeout(300, () => (state.message = ''));
```

Comment:
This did not work while doing QA. Also I'd suggest using the `react-toastify` to render success/error messages.

Reply:
I wasn't using it, I forgot to remove it.
The fix was to remove the success/error messages when there's a change in an input field